### DISCUSSION TITLE
Turn off night mode in settings?

### DISCUSSION THREAD

**Discussion by avestura** [SebLague/Geographical-Adventures#15] [Ideas]
I love that this game supports night mode, but I wish I could disable it in settings sometimes. Comes in handy when I'm just hanging around for learning about shape of earth's surface.
Btw love your work man, this game has so much potential!

**Comment by arrowgent** [comment]
it was almost annoyingly dark.  the only benefit in my playthrough was being able to see the cities

**Comment by SebLague** [comment]
I agree that nighttime can be annoying for navigation. I'll look into improving it, and will also consider adding a permanent day mode to the settings. Thanks for the feedback!
